tests/containers/seccomp.pm

```diff
@@ -0,0 +1,49 @@
+# SUSE's openQA tests
+#
+# Copyright 2024 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Package: seccomp
+# Summary: Test seccomp in docker & podman
+# Maintainer: QE-C team <qa-c@suse.de>
+
+
+use Mojo::Base 'containers::basetest';
+use testapi;
+use serial_terminal qw(select_serial_terminal);
+use utils;
+use containers::common;
+
+sub run {
+    my ($self, $args) = @_;
+    my $runtime = $args->{runtime};
+
+    select_serial_terminal;
+    my $engine = $self->containers_factory($runtime);
+
+    assert_script_run "grep SECCOMP /boot/config-\$(uname -r)";
+    assert_script_run "$runtime info | grep -i seccomp";
+
+    my $image = "registry.opensuse.org/opensuse/bci/bci-busybox:latest";
+    my $policy = "policy.json";
+
+    # Get default policy
+    if ($runtime eq "docker") {
+        script_retry("curl -sLo $policy https://github.com/moby/moby/raw/master/profiles/seccomp/default.json", retry => 5, delay => 60, timeout => 300);
+    } elsif ($runtime eq "podman") {
+        assert_script_run "cp \$(podman info -f '{{ .Host.Security.SECCOMPProfilePath }}' $policy";
```

Comment:
Wouldn't it be better to enforce a static profile that we control, instead of using the one that podman contains itself?

I mean, we should test if a desired state is reached, if this profile would be empty then this case would not be covered.

Reply:
In short: Why not use the `profile.json` from above here as well? :thinking:

Reply:
I wouldn't mix docker & podman profiles.

Reply:
Yes that makes sense. But I would argue that it's better to have a static policy that fits the test run, instead of relying on whatever podman itself provides, for the sake of test stability.

Reply:
Ah, I see this was updated accordingly. Thank you Ricardo!